requirements/requirements.txt

```diff
@@ -1,135 +1,132 @@
-adal==1.2.1               # via msrestazure
-amqp==2.4.2               # via kombu
+adal==1.2.2               # via msrestazure
+amqp==2.5.2               # via kombu
 ansible-runner==1.4.4
 ansiconv==1.0.0
-appdirs==1.4.2
-argparse==1.4.0           # via uwsgitop
 asgi-amqp==1.1.4
 asgiref==1.1.2            # via asgi-amqp, channels, daphne
-asn1crypto==0.24.0        # via cryptography
-attrs==19.1.0             # via automat, service-identity, twisted
-autobahn==19.5.1          # via daphne
-automat==0.7.0            # via twisted
-azure-common==1.1.21      # via azure-keyvault
+attrs==19.3.0             # via automat, jsonschema, twisted
+autobahn==19.11.1         # via daphne
+automat==0.8.0            # via twisted
+azure-common==1.1.23      # via azure-keyvault
 azure-keyvault==1.1.0
 azure-nspkg==3.0.2        # via azure-keyvault
-billiard==3.6.0.0         # via celery
-boto==2.47.0
+billiard==3.6.1.0         # via celery
+boto==2.49.0
 cachetools==3.1.1         # via google-auth
 celery==4.3.0
-certifi==2019.3.9         # via kubernetes, msrest, requests
-cffi==1.12.3              # via cryptography
+certifi==2019.11.28       # via kubernetes, msrest, requests
+cffi==1.13.2              # via cryptography
 channels==1.1.8
 chardet==3.0.4            # via requests
 constantly==15.1.0        # via twisted
-cryptography==2.6.1       # via adal, azure-keyvault, pyopenssl, service-identity
-daphne==1.3.0
-defusedxml==0.5.0         # via python3-openid, python3-saml, social-auth-core
+cryptography==2.8         # via adal, autobahn, azure-keyvault
+daphne==1.4.2
+defusedxml==0.6.0         # via python3-openid, python3-saml, social-auth-core
 dictdiffer==0.8.0         # via openshift
-django-auth-ldap==1.7.0
-django-cors-headers==2.4.0
-django-crum==0.7.2
-django-extensions==2.0.0
+django-auth-ldap==2.1.0
+django-cors-headers==3.2.0
+django-crum==0.7.4
+django-extensions==2.2.5
 django-jsonfield==1.2.0
 django-oauth-toolkit==1.1.3
-django-pglocks==1.0.2
-django-polymorphic==2.0.2
+django-pglocks==1.0.3
+django-polymorphic==2.1.2
 django-qsstats-magic==1.1.0
 django-radius==1.3.3
 django-solo==1.1.3
-django-split-settings==0.3.0
-django-taggit==0.22.2
-django==2.2.4
+django-split-settings==1.0.0
+django-taggit==1.2.0
+django==2.2.7
```

Comment:
👀 https://github.com/advisories/GHSA-hvmf-r92r-27hr

(we should go with 2.2.8)